servers/audio/effects/audio_effect_spectrum_analyzer.cpp

```diff
@@ -81,9 +81,10 @@ void AudioEffectSpectrumAnalyzerInstance::process(const AudioFrame *p_src_frames
 
 		float *fftw = temporal_fft.ptrw();
 		for (int i = 0; i < to_fill; i++) { //left and right buffers
-			fftw[(i + temporal_fft_pos) * 2] = p_src_frames[i].l;
+			float window = -.5 * Math::cos(2. * Math_PI * (double)i / (double)to_fill) + .5;
```

Comment:
Could you use fully qualified floats, i.e. `0.5` and `2.0`? That makes code more readable IMO and is the usual way we write them in this codebase.